lhotse/features/kaldi/extractors.py

```diff
@@ -76,6 +76,10 @@ def device(self) -> Union[str, torch.device]:
     def frame_shift(self) -> Seconds:
         return self.config.frame_shift
 
+    def to(self, device: str):
+        self.config.device = device
+        self.extractor.to(device)
```

Comment:
Because I would like to instantiate extractors using `.from_yaml(...)` from the same config file, but use them on CPUs and GPUs.